docs/network/smart-rollup-nodes.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Running an Etherlink Smart Rollup node
+---
+
+Etherlink uses Smart Rollup nodes as a bridge between the EVM transactions on the EVM nodes and Tezos layer 1.
+Public Smart Rollup nodes for Etherlink are not yet available, so you must run your own if you want to participate in the Etherlink network.
+
+The Smart Rollup node runs the _kernel_, which is a Rust program compiled in WASM implementing the semantics of Etherlink blocks and transactions.
+
+To start, the node uses the _installer kernel_, which is a compressed version of the kernel that provides only enough information to install the original kernel.
+The data for the original kernel is stored in separate files called _preimages_.
+
+## References
+
+Make sure that you understand the interaction between different nodes as described in [Etherlink architecture](./architecture).
+
+For more information about Smart Rollup nodes in general, see [Smart Rollups](https://docs.tezos.com/architecture/smart-rollups) on docs.tezos.com and [Smart Rollup Node](https://tezos.gitlab.io/shell/smart_rollup_node.html) in the Octez documentation.
+
+## Running the Smart Rollup node from a snapshot
+
+You can start the Smart Rollup node with a snapshot of the Etherlink state to prevent it from having to compute the state from Etherlink genesis.
+
+For simplicity, these steps show how to run the smart rollup node in observer mode:
+
+1. Get a built version of the Smart Rollup node binary, named `octez-smart-rollup-node`.
+The best place to get the most recent binary files to use with Etherlink is https://gitlab.com/tezos/tezos/-/releases.
+
+1. Initialize the local context of the node, which is where it stores local data:
+
+   1. Set the environment variable `sr_observer_data_dir` to the directory where the node should store its local data.
+   The default value is `$HOME/.tezos-smart-rollup-node`.
+   1. Initialize the local context by running this command:
+
+      ```bash
+      octez-smart-rollup-node init observer config for sr1Ghq66tYK9y3r8CC1Tf8i8m5nxh8nTvZEf \
+        with operators --data-dir $sr_observer_data_dir \
+        --pre-images-endpoint https://snapshots.eu.tzinit.org/etherlink-mainnet/wasm_2_0_0
```

Comment:
Shouldn’t we add the same paragraph about relying on this service vs. building the preimages yourself? now that they are not in the same page.

Reply:
Oh you did that, my bad.